src/Integrations/Integrations/PDO/PDOIntegration.php

```diff
@@ -14,6 +14,21 @@ class PDOIntegration extends Integration
 
     const CONNECTION_TAGS_KEY = 'connection_tags';
 
+    const DB_DRIVER_TO_SYSTEM = [
+        'cubrid' => 'other_sql',
+        'dblib' => 'other_sql',
+        # may be mssql or Sybase, not supported anymore so shouldn't be a problem
```

Comment:
Comment style should be `//` or `/* */` these days, not `#`, which have been repurposed for attributes.

Reply:
fixed

Reply:
(Using # followed by a space is and will always be fine.)